src/web/views/locale/en/confirm.html

```diff
@@ -0,0 +1,16 @@
+<p class="govuk-body">Please keep a copy of this page for your records.</p>
+
+<h2 class="govuk-heading-l">What happens next</h2>
+<h3 class="govuk-heading-m">1. Your prepaid card will be sent</h3>
+<p class="govuk-body">We will send your prepaid card to the address you gave us.</p>
+<h3 class="govuk-heading-m">2. Money will be added to your card</h3>
+<p class="govuk-body">We will add money to your card every 4 weeks so that you can buy healthy food and milk. You will get £12.40 every 4 weeks until your baby is born.</p>
+<h3 class="govuk-heading-m">3. You can use your card to pay for fruit, vegetables and milk</h3>
+<p class="govuk-body">You can find out <a class="govuk-link" href="related/how-to-buy-with-prepaid-card">how to use your prepaid card and what you can buy with your money on GOV.UK (link opens in a new tab)</a></p>
+<div class="govuk-warning-text">
+    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+    <strong class="govuk-warning-text__text">
+        <span class="govuk-warning-text__assistive">Warning</span>
+        When your baby is born you must telephone XXX to tell us. We will then increase your money to £6.20 each week until your baby is 1 year old.
```

Comment:
I think Brian has a support telephone now - he should be able to provide the number

Reply:
The comment from Brian is that we should leave it this way for now as we don't know what this should be and hence should leave it like this so that its obvious we need to revisit this page.